onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -416,7 +413,7 @@ def aten_leaky_relu_backward(
 
 
 @torch_op("aten::linear")
-def aten_linear(input, weight, bias=None) -> TensorType:
+def aten_linear(input: TFloat, weight: TFloat, bias: Optional[TFloat] = None) -> TensorType:
```

Comment:
The result type of both MatMul and Add belongs to TFloat, should we change the TensorType to TFloat?

Reply:
Good catch! Done.